cumulusci/cli/cci.py

```diff
@@ -1419,3 +1395,71 @@ def flow_run(runtime, flow_name, org, delete_org, debug, o, skip, no_prompt):
                 "Scratch org deletion failed.  Ignoring the error below to complete the flow:"
             )
             click.echo(str(e))
+
+
+@error.command(name="info", help="Outputs the last part of the most recent traceback")
+@click.option("--num-lines", "-n", default=30, show_default=True, type=int)
+def info(num_lines):
+    if not CCI_LOGFILE_PATH.is_file():
+        click.echo(f"No logfile found at: {CCI_LOGFILE_PATH}")
+    else:
+        output = lines_from_traceback(CCI_LOGFILE_PATH.read_text(), num_lines or 30)
```

Comment:
If `num_lines` isn't passed by the user then it should be equal to the default value, 30.